src/js/edu-benefits/containers/veteran-information/VeteranAddressPage.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```

Comment:
Not really relevant to this PR in particular, but I think all of our Page components are the same with just the Fields component being different between them. Probably could create a wrapper that we reuse.